src/aarch64/Gstep.c

```diff
@@ -42,7 +42,7 @@ static const int WSIZE = sizeof (unw_word_t);
 static int
 is_plt_entry (struct dwarf_cursor *c)
 {
-  unw_word_t w0, w1;
+  unw_word_t w0 = 0, w1 = 0;
```

Comment:
Warning was about w1 being uninitialized on line 142. I set both w0 and w1 for consistency.